### ISSUE TITLE
[Bug?] Relationships involving a one-to-one relationship throw an exception

### ISSUE THREAD

**Issue by EPadronU** [coleifer/peewee#674]
## Sample layout

```
class User(database.Model):
    pass


class Patient(database.Model):
    user = ForeignKeyField(User, primary_key=True)


class Disease(database.Model):
    pass


class PatientDisease(database.Model):
    patient = ForeignKeyField(Patient)
    disease = ForeignKeyField(Disease)
```

## The code that throws the exception

```
>> patient = Patient.get(user=1)
>> patient.patientdisease_set
```

## Output | head -n 6

```
$PWD/.venv/lib/python2.7/site-packages/peewee.pyc in _parse(self, node, alias_map, conv)
   1546             if conv and isinstance(conv, ForeignKeyField):
   1547                 params = [
-> 1548                     conv.to_field.db_value(getattr(node, conv.to_field.name))]
   1549             else:
   1550                 params = [node._get_pk_value()]

AttributeError: 'User' object has no attribute 'user'
```

## Observations

- This also happens with one-to-many relationships
- This not happens in the other way. For example: `patient_disease.patient`

## Pip freeze

```
blinker==1.3
Flask==0.10.1
Flask-Login==0.2.11
Flask-Mail==0.9.1
Flask-Markdown==0.3
Flask-Principal==0.4.0
Flask-Security==1.7.4
Flask-WTF==0.12
ipython==3.2.1
itsdangerous==0.24
Jinja2==2.7.3
Markdown==2.6.2
MarkupSafe==0.23
passlib==1.6.2
peewee==2.6.3
Werkzeug==0.10.4
wheel==0.24.0
wtf-peewee==0.2.3
WTForms==2.0.2
```

**Comment by coleifer** [comment]
Thanks for reporting, fixed in c1030f58c0df67b28fefa75182d6a93698921e01

**Comment by EPadronU** [comment]
Thanks for fixing it so quickly. Cheers.
